src/paths.config.js

```diff
@@ -0,0 +1,5 @@
+export default {
+	api: {
+		authGoogle: 'http://localhost:9000/auth/google'
```

Comment:
interesting approach :) imho it's better to specify `baseUrl = http://localhost:9000` for axios, then you can just use /auth/google in component